doc/_themes/mantella/layout.html

```diff
@@ -26,7 +26,7 @@
       {{ toctree() }}
     </nav>
     <main>
-      <aside>
+      <aside id="breadcrumbs">
```

Comment:
I understand your reason for using a unambiguous name instead of `aside`.

However, relying only on the id (or class) within the CSS side removes some information on the documents structure, as `#breadcrumbs` could be defined at any document level.

So instead of
```
.breadcrumbs 
```
using
```
main aside.breadcrumbs {
```
feels more precise.